holoviews/ipython/__init__.py

```diff
@@ -152,6 +152,7 @@ def __call__(self, *args, **params):
             if p in self._backends:
                 imports.append((p, self._backends[p]))
         if not imports:
+            args = ['matplotlib']
```

Comment:
Yes, this was a bug, matplotlib would be imported but ``MPLRenderer.load_nb`` was never called.